force-app/main/default/objects/Account/validationRules/EDRD_PhoneNumber_Validation.validationRule-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>EDRD_PhoneNumber_Validation</fullName>
+    <active>true</active>
+    <errorConditionFormula>OR(
```

Comment:
@deepakmulamalla Is this validation rule intended to apply to all accounts (SAT and EDRD)?

Reply:
@cgijeffolsen, I’ve updated the validation rule to include the EDRD user role check. Please review, and let me know if there are any further adjustments needed.